src/ai/susi/server/api/aaa/LoginService.java

```diff
@@ -298,9 +298,9 @@ else if(pubkeyLogin){ // second part of pubkey login: verify if the response to
 				sig.update(challenge.getBytes());
 				verified = sig.verify(Base64.getDecoder().decode(challangeResponse));
 			} catch (NoSuchAlgorithmException e){
-				throw new APIException(400, "No such algorithm");
```

Comment:
I think here should also not be a 404 because the parameter are wrong but the route exist